geoportal/c2cgeoportal_geoportal/scaffolds/create/docker-compose.override.sample.yaml_tmpl

```diff
@@ -13,11 +13,15 @@ services:
       # For Python project development.
       - ${PWD}/geoportal/{{package}}_geoportal:/app/{{package}}_geoportal
       # For Python c2cgeportal development.
-      # - '${PWD}/../c2cgeoportal:/opt/c2cgeoportal'
+      # - '${PWD}/../c2cgeoportal/commons/c2cgeoportal_gcommons:/opt/c2cgeoportal/commons/c2cgeoportal_commons'
+      # - '${PWD}/../c2cgeoportal/geoportal/c2cgeoportal_geoportal:/opt/c2cgeoportal/geoportal/c2cgeoportal_geoportal'
+      # - '${PWD}/../c2cgeoportal/admin/c2cgeoportal_admin:/opt/c2cgeoportal/admin/c2cgeoportal_admin'
     environment:
       - GUNICORN_CMD_ARGS=--reload
       - C2CWSGIUTILS_CONFIG=/app/development.ini
       # - PRINT_URL=http://print:8080/print/
+    ports:
+      - 5678:5678
```

Comment:
- 5678:5678  # If you are using Visual Studio Code
In my case Eclipse is listen to port 5678, and the python process connect to 172.17.0.1:5678

Reply:
I don't understand your comment, finally it's the same, not?

Reply:
It is the inverse:
With VS Code: Python process listen port 5678, and VS Code is the client.
But Eclipse (or IntelliJ, PyCharm) listen port 5678 and python process is the client.

Reply:
OK, thanks :-)